tests/integration/data_sources_and_expectations/test_canonical_expectations.py

```diff
@@ -154,3 +154,14 @@ def test_expect_column_mean_to_be_between(batch_for_datasource):
     expectation = gxe.ExpectColumnMeanToBeBetween(column="a", min_value=2, max_value=3)
     result = batch_for_datasource.validate(expectation)
     assert result.success
+
+
+@parameterize_batch_for_data_sources(
+    data_source_configs=SQL_DATA_SOURCES,
+    data=pd.DataFrame({"a": ["Chicago", "chicago", "CHICAGO"]}),
+)
+def test_expect_column_value_to_match_like_pattern(batch_for_datasource):
```

Comment:
We have a dedicated file for testing this type of expectation now. So this should be moved over there. But does this prove anything those tests didn't prove? Looks like I only had a subset of data sources running, so maybe that's the difference?

It's hard to tell that this works based on the integration test since the issue is so low-level. I'd consider also adding a unit test around the method you updated that mocks out the existence of those deps.

Reply:
yes, that's a great idea - unit test added here: [e290b2e](https://github.com/great-expectations/great_expectations/pull/10745/commits/e290b2ecd5aec3809e4c9029a14e15292485a304)